docs/server-side-rendering.md

```diff
@@ -22,26 +22,11 @@ React components used on the server will be rendered to HTML by being passed to
 
 ### Caching
 
-Because it is necessary to serve the redux state along with a server-rendered page, we use two levels of cache on the server: one to store the redux state, and one to store rendered layouts.
+Because it is necessary to serve the redux state along with a server-rendered page, we use two levels of cache on the server: one to [store the redux state](https://github.com/Automattic/wp-calypso/blob/master/server/state-cache/index.js), and one to [store rendered layouts](https://github.com/Automattic/wp-calypso/blob/7ded1642a5b95a30b64fe0a3e462ddd2317c0df2/server/render/index.js#L33).
 
-##### Data Cache
+Both caches use the same key, which is the pathname of the URL. URLs with query args are not cached unless the arg name is present in `context.queryCacheKeys`, in which case the argument or arguments are appended to the key.
 
-At render time, the Redux state is [serialized and cached](../server/render/index.js), using the current path as the cache key, unless there is a query string, in which case we don't cache.
```

Comment:
We could retain a modified version of this (keys are now generated from a combination of path and whitelisted qargs).

Reply:
I think we have this covered above.